.babelrc

```diff
@@ -3,11 +3,12 @@
     "react"
   ],
   "plugins": [
-    "transform-es2015-modules-amd",
```

Comment:
Now it's done by Rollup (but I still use this plugin when compile app for unit tests)